transport/rif_comms/node.py

```diff
@@ -303,11 +298,44 @@ def log(self):
         return self._log
 
     def __repr__(self):
-        node = f" node:{pex(self._raiden_service.address)}" if self._raiden_service else ""
+        node = f" RIF Comms Transport node:{pex(self.address)}"
         return f"<{self.__class__.__name__}{node} id:{id(self)}>"
 
 
 class LightClientNode(Node):
 
-    def __init__(self, address: Address, config: dict, auth_params: dict):
+    def __init__(self, address: Address, config: dict):
         Node.__init__(self, address, config)
+
+    def _handle_message(self, message: RaidenMessage):
+        """
+        Handle received Raiden message.
+        """
+        if self.stop_event.ready():
+            return  # ignore when node is stopped
+
+        # process message if its type is expected
+        if isinstance(message, (Delivered, Processed, SignedRetrieableMessage)):
+            self.log.info(
+                "Raiden message received",
+                type=type(message),
+                node=pex(self.address),
+                message=message,
+                sender=pex(message.sender),
+            )
+            # Pass message to raiden service for business logic. The message will be stored on the HUB database.
+            self._raiden_service.on_message(message, True)
+        else:
+            self.log.warning(
+                "unexpected type of message received",
+                type=type(message),
+                node=pex(self.address),
+                message=message,
+            )
+
+    def _ack_message(self, message: (Processed, SignedRetrieableMessage)):
+        """
+        Acks must be signed by the Light client first, therefore the transport is not in charge to create and send
+        the Delivered messages.
+        """
+        raise Exception("Do not use _ack_message for light client transport")
```

Comment:
this is nice and makes sense, but i am assuming the signed `Delivered` message will be sent somewhere after the  `self._raiden_service.on_message` call on line `327`?

Reply:
Exactly, the LCSDK has the responsability of sending the acks